apps/ca.c

```diff
@@ -82,7 +82,9 @@
 #define ENV_DATABASE            "database"
 
 /* Additional revocation information types */
+typedef int REVINFO_TYPE;
```

Comment:
Well, we don’t like enum’s in public API’s for the reasons you state.  But within openssl itself it’s less of an issue.